### DISCUSSION TITLE
I/O işlemleri dışında herhangi bir kütüphane kullanımı.

### DISCUSSION THREAD

**Discussion by essteec** [gusanmaz/bmb212-algorithms-2024#5] [Q&A] [answered]
![resim](https://github.com/gusanmaz/bmb212-algorithms-2024/assets/71087683/6703e5bf-79bb-44f6-8efe-65668f3d0032)
Burada verilen fonksiyonun List döndürmesi gerekiyor. Bunun için de javada java.util.ArrayList ve java.util.List kütüphanelerini kullanmamız gerekiyor. Buna benzer durumlarda sadece gerekli olan kütüphaneleri kullanabilir miyiz?

Bu arada aynı görselde yandaki Max Consecutive Ones II de sadece premium kullanıcılarının erişimine açık.

**Comment by gusanmaz** [comment, marked as answer]
Evet, 

- bu gibi durumlarda gerekli kutuphaneleri kullanabilirsiniz. Bunun gibi ek kutuphane kullanmanizi gerektirecek sorulari isterseniz atlayabilirsiniz de.
- Sadece Premium'a acik sorulari gondermeniz istenmemektedir.

Kolay gelsin,
